smoked sheep/Enemy.cpp

```diff
@@ -0,0 +1,268 @@
+	{ 0.0f,  64.0f, 0.5f, 1.0f, 0xFFFFFFFF, 0.0f, 1.0f },
+};
+enemy_wolf wolf[100];
+
+float Enemy_tu(enemytype a) {
+	switch (a)
+	{
+	case nomal:
+		return 0.f / 2048.f;
+	case speed:
+		return 800.f / 2048.f;
+	case strong:
+		return 0.f / 2048.f;
+	case hide:
+		return 0.f / 2048.f;
+	}
+}
+
+float Enemy_tv(enemytype a) {
+	switch (a)
+	{
+	case nomal:
+		return 0.f / 2048.f;
+	case speed:
+		return 0.f / 2048.f;
+	case strong:
+		return 287.f / 2048.f;
+	case hide:
+		return 574.f / 2048.f;
+	}
+}
+
+float EnemyWidth_tu() {
+	return 200.f / 2048.f;
+}
+float EnemyHeight_tv() {
+
+	return 287.f / 2048.f;
+}
+void FormatEnemy_tu()
+{
+	enemy[enemy_number][0].tu = 0.f / 2048.f;
+	enemy[enemy_number][1].tu = EnemyWidth_tu();
+	enemy[enemy_number][2].tu = EnemyWidth_tu();
+	enemy[enemy_number][3].tu = 0.f / 2048.f;
+}
+void FormatEnemy_tv() {
+	enemy[enemy_number][0].tv = 0.f / 2048.f;
+	enemy[enemy_number][1].tv = 0.f / 2048.f;
+	enemy[enemy_number][2].tv = EnemyHeight_tv();
+	enemy[enemy_number][3].tv =  EnemyHeight_tv();
+}
+
+void SettingEnemy_tv(enemytype a)
+{
+	FormatEnemy_tv();
+	for (int i = 0; i < 4; i++)
+	{
+		enemy[enemy_number][i].tv += Enemy_tv(a);
+	}
+}
+
+void SettingEnemy_tu(enemytype a) {
+	FormatEnemy_tu();
+	for (int i = 0; i < 4; i++)
+	{
+		enemy[enemy_number][i].tu += Enemy_tu(a);
+	}
+}
+
+void SettingEnemy_tutv(enemytype a)
+{
+	
+		SettingEnemy_tv(a);
+		SettingEnemy_tu(a);
+	
+}
+
+//
+//float Enemy_tu(enemytype a) {
+//	switch (a)
+//	{
+//	case nomal:
+//		return 125.f / 1024.f;
+//	case speed:
+//		return 170.f / 1024.f;
+//	case strong:
+//		return 130.f / 1024.f;
+//	case hide:
+//		return 200.f / 1024.f;
+//	}
+//}
+//
+//void SettingEnemy_tu()
+//{
+//	enemy[enemy_number][0].tu = 0.f / 1024.f;
+//	enemy[enemy_number][1].tu = Enemy_tu(wolf[enemy_number].type);
+//	enemy[enemy_number][2].tu = Enemy_tu(wolf[enemy_number].type);
+//	enemy[enemy_number][3].tu = 0.f / 1024.f;
+//}
+//
+//
+//void SettingEnemy_tv(enemytype a)
+//{
+//	switch (a)
+//	{
+//	case nomal:
+//		enemy[enemy_number][0].tv = 0.f / 1024.f;
+//		enemy[enemy_number][1].tv = 0.f / 1024.f;
+//		enemy[enemy_number][2].tv = 140.f / 1024.f;
+//		enemy[enemy_number][3].tv = 140.f / 1024.f;
+//		break;
+//	case speed:
+//		enemy[enemy_number][0].tv = 140.f / 1024.f;
+//		enemy[enemy_number][1].tv = 140.f / 1024.f;
+//		enemy[enemy_number][2].tv = 240.f / 1024.f;
+//		enemy[enemy_number][3].tv = 240.f / 1024.f;
+//		break;
+//	case strong:
+//		enemy[enemy_number][0].tv = 240.f / 1024.f;
+//		enemy[enemy_number][1].tv = 240.f / 1024.f;
+//		enemy[enemy_number][2].tv = 385.f / 1024.f;
+//		enemy[enemy_number][3].tv = 385.f / 1024.f;
+//		break;
+//	case hide:
+//		enemy[enemy_number][0].tv = 510.f / 1024.f;
+//		enemy[enemy_number][1].tv = 510.f / 1024.f;
+//		enemy[enemy_number][2].tv = 655.f / 1024.f;
+//		enemy[enemy_number][3].tv = 655.f / 1024.f;
+//		break;
+//	}
+//}
+
+
+void EnemyInit(int type,int enemy_number) {
```

Comment:
関数は処理なので関数名は基本的に動作が主体になるようにしたほうがいいです
```suggestion
void InitEnemy(int type,int enemy_number) {
```
あと初期化という抽象的な名前を使うよりは生成などのほうが分かりやすい気がします